src/components/algorithm/Algorithms.js

```diff
@@ -18,6 +18,7 @@ import { getAlgorithms, deleteAlgorithm } from '../../actions';
 import Loader from '../common/Loader';
 import Table from '../common/Table';
 import { buildEditAlgorithmPath, ADD_ALGORITHM_PATH } from '../../config/paths';
+import { AUTHOR_GRAASP, AUTHOR_USER } from '../../config/constants';
```

Comment:
my linter alerted me to this: `AUTHOR_GRAASP is defined but never used`. not sure why yours didn't catch it!